packages/headless/src/features/search/search-request.ts

```diff
@@ -100,13 +101,28 @@ function getAllEnabledFacets(state: StateNeededBySearchRequest) {
 
 function getAllFacets(state: StateNeededBySearchRequest) {
   return [
-    ...getFacetRequests(state.facetSet ?? {}),
+    ...getSpecificFacetRequests(state.facetSet ?? {}),
     ...getRangeFacetRequests(state.numericFacetSet ?? {}),
     ...getRangeFacetRequests(state.dateFacetSet ?? {}),
     ...getFacetRequests(state.categoryFacetSet ?? {}),
   ];
 }
 
+function getSpecificFacetRequests<T extends FacetSetState>(state: T) {
+  return getFacetRequests(state).map((request) => {
+    /* The Search API does not support 'alphanumericDescending' as a string value and instead relies on a new sort mechanism to specify sort order.
```

Comment:
Have you created the corresponding ticket in the V3 epic for UI kit ?

Reply:
Yup 🙂 
https://coveord.atlassian.net/browse/KIT-2578